gnome-initial-setup/pages/account/gis-account-page-local.c

```diff
@@ -388,11 +391,32 @@ confirm (GisAccountPageLocal *page)
     g_signal_emit (page, signals[CONFIRM], 0);
 }
 
+static gchar *
+get_default_avatar_image (void)
+{
+  GisDriver *driver;
+  gchar *default_avatar;
+
+  driver = GIS_DRIVER (g_application_get_default ());
```

Comment:
```suggestion
  driver = GIS_PAGE (self)->driver;
```

is a bit less obscure.

Reply:
Unfortunately I can't do that. This is `GisAccountPageLocal` which is not a `GisPage`.

Reply:
Good point, well made.